app/src/androidTest/java/com/github/polypoly/app/base/game/GameTest.kt

```diff
@@ -146,4 +147,54 @@ class GameTest {
                 }
         }
     }
+
+    @Test
+    fun whenNextTurnComputesTransactions() {
+
+        gameLobby.start()
+        val game = Game.gameInProgress!!
```

Comment:
`gameLobby.start()` returns the game created that you can use instead of `Game.gameInProgress!!`

Reply:
you are right thanks :) !